api/gas/index.js

```diff
@@ -117,14 +77,41 @@ module.exports = (stores, useCases) => {
 
   });
 
+  app.put('/gasStations/:gasStationId', middlewares.auth, function(req, res) {
+    const useCase = new useCases.HostessToggleGasStation({
+      branchId: req.params.gasStationId,
+      hostessId: config.entities.hostess || req.user.sub,
+      hostessStore: stores.hostessStore,
+      branchStore: stores.branchStore,
+    });
+
```

Comment:
Shouldn't you do a modelToObject before sending right here? 

This returns 🤢 
```
{
    "_id": "5c49d5f785d72c75de5f4da5",
    "_name": "Branch Seed",
    "_address": "Branch Seed Address",
    "_coordinates": [
        10,
        10
    ],
    "_restaurant": {
        "_id": "5c49d5f785d72c75de5f4da4"
    },
    "waitingTurns": 0
}
```

Reply:
That's the object. Take a look at the `Branch` implementation, you will see that most of the properties are private. Sorry for the missing `serializer` class. Maybe if you would have helped 2 weeks ago with this.. HDTPM